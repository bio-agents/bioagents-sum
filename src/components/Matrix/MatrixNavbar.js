import React from 'react'
import { Image, Navbar } from 'react-bootstrap'
import iechorLogo from '../../images/iechor-logo.png'
import NavbarForm from '../Services/NavbarForm'
import { allowCollectionChange } from '../../bioagentsSum/common/helperFunctions'

const MatrixNavbar = () => (
  <Navbar collapseOnSelect bsStyle='default'>
    <Navbar.Header>
      <Navbar.Brand>
        <Image src={iechorLogo} responsive style={{ width: '112px', height: '73px', marginTop: '-10px', marginBottom: '-15px' }} />
      </Navbar.Brand>
      <Navbar.Toggle />
    </Navbar.Header>
    <Navbar.Collapse>
      {allowCollectionChange &&
        <Navbar.Form pullRight>
          <NavbarForm />
        </Navbar.Form>
      }
    </Navbar.Collapse>
  </Navbar>
)

export default MatrixNavbar
