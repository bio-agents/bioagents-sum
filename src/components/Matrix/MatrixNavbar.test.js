import React from 'react'
import renderer from 'react-test-renderer'
import MatrixNavbar from './MatrixNavbar'

it('Renders MatrixNavbar correctly', () => {
  const component = renderer.create(
    <MatrixNavbar />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
