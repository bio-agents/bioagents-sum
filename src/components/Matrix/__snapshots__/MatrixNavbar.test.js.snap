exports[`test Renders MatrixNavbar correctly 1`] = `
<nav
  className="navbar navbar-default">
  <div
    className="container">
    <div
      className="navbar-header">
      <img
        className="navbar-brand img-responsive"
        src="iechor-logo.png"
        style={
          Object {
            "height": "73px",
            "marginBottom": "-15px",
            "marginTop": "-10px",
            "width": "112px",
          }
        } />
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button">
        <span
          className="sr-only">
          Toggle navigation
        </span>
        <span
          className="icon-bar" />
        <span
          className="icon-bar" />
        <span
          className="icon-bar" />
      </button>
    </div>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse" />
  </div>
</nav>
`;
